misk-admin/src/main/kotlin/misk/web/metadata/config/ConfigMetadataAction.kt

```diff
@@ -14,7 +14,7 @@ import misk.web.mediatype.MediaTypes
 class ConfigMetadataAction @Inject constructor(
   private val metadataProvider: Provider<ConfigMetadata>
 ) : WebAction {
-  @Get("/api/config/metadata")
+  @Get("/api/v1/config/metadata")
```

Comment:
To maintain compatibility, both the old Misk Web UI still relying on this and the web actions have been migrated to a v1 prefix. This prevents and conflict with the new filterable `AllMetadataAction`